### PR TITLE
feat: allow *.csproj files

### PR DESCRIPTION
**Pull request by lili2311**
- [x] Ready for review
- [x] Follows [CONTRIBUTING](https://github.com/snyk/broker/blob/master/.github/CONTRIBUTING.md) rules

#### What does this PR do?
- Allow a new manifest type `*.csproj` for .NET projects

#### Any background context you want to provide?
This is part of the planned work around .net & .net core SCM support

### CODE REVIEW

File: client-templates/github-com/accept.json.sample [hunk 1]
```diff
@@ -90,6 +90,14 @@
...
             "path": "commits.*.modified.*",
             "value": "packages.config"
+          },
+          {
+            "path": "commits.*.added.*",
+            "value": "*.csproj"
+          },
+          {
+            "path": "commits.*.modified.*",
+            "value": "*.csproj"
```
Comment (adrukh) [inline client-templates/github-com/accept.json.sample]: I wonder if the filter validation treats `*` as wildcards in the `value` field as well.

File: client-templates/gitlab/accept.json.sample [hunk 2]
```diff
@@ -118,7 +124,8 @@
             "**/build.gradle",
             "**/build.sbt",
             "**/.snyk",
-            "**/packages.config"
+            "**/packages.config",
+            "**/*.csproj"
```
Comment (adrukh) [inline client-templates/gitlab/accept.json.sample]: Same wildcard question here.

File: client-templates/github-com/accept.json.sample (hunk 1 above)
Comment (lili2311) [reply client-templates/github-com/accept.json.sample]: testing locally, will confirm shortly

Comment (lili2311) [reply client-templates/github-com/accept.json.sample]: Verified locally, all events looked good nothing blocked when testing with local webhooks setup.

**Comment by lili2311** [comment]
https://snyksec.atlassian.net/browse/SC-6163

**Pull request merged**